extensions-core/namespace-lookup/src/test/java/io/druid/query/extraction/namespace/NamespacedExtractorTest.java

```diff
@@ -1,201 +0,0 @@
-import java.util.Collections;
-import java.util.List;
-import java.util.UUID;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
-
-/**
- *
- */
-public class NamespacedExtractorTest
-{
-  private static final ConcurrentMap<String, Function<String, String>> defaultMap = new ConcurrentHashMap<>();
-  private static final Function<String, String> NOOP_FN = new Function<String, String>()
-  {
-    @Nullable
-    @Override
-    public String apply(@Nullable String input)
-    {
-      return Strings.isNullOrEmpty(input) ? null : input;
-    }
-  };
-
-  private static final Function<String, List<String>> NOOP_REVERSE_FN = new Function<String, List<String>>()
-  {
-    @Nullable
-    @Override
-    public List<String> apply(@Nullable String input)
-    {
-      return Strings.isNullOrEmpty(input) ? Collections.<String>emptyList() : Arrays.asList(input);
-    }
-  };
-
-  private static final Function<String, Function<String, String>> defaultFnFinder = new Function<String, Function<String, String>>()
-  {
-    @Nullable
-    @Override
-    public Function<String, String> apply(@Nullable String input)
-    {
-      Function<String, String> fn = defaultMap.get(input);
-      return fn == null ? NOOP_FN : fn;
-    }
-  };
-
-  private static final Function<String,Function<String, List<String>>> defaultReverseFnFinder = new Function<String, Function<String,List<String>>>()
-  {
-    @Nullable
-    @Override
-    public Function<String, java.util.List<String>> apply(@Nullable final String value)
-    {
-      return NOOP_REVERSE_FN;
-    }
-  };
-
-  @BeforeClass
-  public static void setupStatic()
-  {
-    defaultMap.put(
-        "noop", new Function<String, String>()
-        {
-          @Nullable
-          @Override
-          public String apply(String input)
-          {
-            return input;
-          }
-        }
-    );
-    defaultMap.put(
-        "null", new Function<String, String>()
-        {
-          @Nullable
-          @Override
-          public String apply(@Nullable String input)
-          {
-            return null;
-          }
-        }
-    );
-    defaultMap.put(
-        "turtles", new Function<String, String>()
-        {
-          @Nullable
-          @Override
-          public String apply(@Nullable String input)
-          {
-            return "turtle";
-          }
-        }
-    );
-    defaultMap.put(
-        "empty", new Function<String, String>()
-        {
-          @Nullable
-          @Override
-          public String apply(@Nullable String input)
-          {
-            return "";
-          }
-        }
-    );
-  }
-
-  @Test
-  public void testSimpleNamespace()
-  {
-    NamespacedExtractor namespacedExtractor = new NamespacedExtractor(defaultFnFinder, defaultReverseFnFinder, "noop");
-    for (int i = 0; i < 10; ++i) {
-      final String val = UUID.randomUUID().toString();
-      Assert.assertEquals(val, namespacedExtractor.apply(val));
```

Comment:
can we replace those tests instead of removing them. it is important to test apply and unapply